backend/app/services/filter_service.rb

```diff
@@ -9,7 +9,7 @@ def is_valid_key(key, table_name)
     when "users"
       %w[email].include? key
     when "enrollments"
-      %w[id siret nom_raison_sociale target_api status team_members.email only_with_unprocessed_messages].include? key
+      %w[id siret nom_raison_sociale target_api status zip_code team_members.email only_with_unprocessed_messages].include? key
```

Comment:
zipcode filter is not working ? 

https://user-images.githubusercontent.com/43042737/223136098-82e6c945-b63d-4667-9249-f57aa9123199.mov

Reply:
There is also a warning in the `user list` 

Is it the same for you ? 
<img width="757" alt="Capture d’écran 2023-03-06 à 15 25 22" src="https://user-images.githubusercontent.com/43042737/223137744-d2faded9-0dd2-44a7-9aff-02afa86a8deb.png">